projects/frontend/cicd/build_data_pipelines.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+###
+# For installing the build prerequisites, consider `install_data_pipelines.sh` script.
+# This script is meant to be used, to quickly
+# rebuild and link the the @vdk/data-pipelines in an already bootstrapped development setup.
+###
+
+if ! which npm >/dev/null 2>&1 ; then
+  echo "ERROR:"
+  echo "Please install npm 8.5.5+. Build cannot continue without it."
+  exit 1
+fi
+echo "Logging npm engines version..."
+npm version
+
+cd "$(dirname $0)" || exit 1
```

Comment:
Since we're treating `data-pipelines` and 'shared-components` as separate packages, maybe we should split the CI into the subfolders instead.

Reply:
Those npm artifacts separated, are something to push faster and unblock adoption. We will next be working on having a potentially image, that contains the frontend installation, so the Frontend component (the way described VEP-1507), is going to be delivered at once to users
And the compounds of the Frontend could be also consumed;

So, in that sense, and also considering the reuse of commons in gitlab-ci.yml between subprojects, for example change locations to trigger pipeline jobs, I still think it's good to continue having common `cicd/` dir for the Frontend component.